scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/OpDescription.java

```diff
@@ -46,8 +50,8 @@ public final class OpDescription {
      * @param info the {@link OpInfo} of interest
      * @return a descriptor for this {@link OpInfo}
      */
-    public static String basic(final OpInfo info) {
-        return basic(info, null);
+    public static String verbose(final OpInfo info) {
+        return verbose(info, null);
```

Comment:
I kind of wonder if this  class should just be combined with `SimplifiedOpDescriptionGenerator` somehow...